crates/tabby-scheduler/src/code/index.rs

```diff
@@ -88,7 +88,7 @@ fn add_changed_documents(cache: &mut CacheStore, repository: &RepositoryConfig,
         }
 
         indexed_files_batch
-            .set(&file_id, &String::new())
+            .set(&file_id, &INDEX_VERSION.to_string())
```

Comment:
add a method in cache, e.g `Cache::set_indexed(&indexed_files_batch, &file_id)`

Reply:
We had this previously, and you requested it be changed to a batched method. I'm going to make a wrapper type for `Bucket` which handles this batching instead.

Reply:
this is for avoiding leaking implementation details (algorithm version) to places beyond cache.

Why a wrapper type is needed, cannot `set_indexed` takes `&Batch` as input (as the suggested signature shows)?

Reply:
If we're taking a `Batch`, then we will need to be setting the values on that batch. By using a wrapper type it can automatically use the `INDEX_ALGORITHM_VERSION` constant without exposing it to `index.rs`